modules/ROOT/pages/apikit-tutorial.adoc

```diff
@@ -24,7 +24,7 @@ link:{attachmentsdir}/api.raml[api.raml]
 +
 . Check Add APIkit Components. In the RAML File field, browse to and select the downloaded `api.raml` file.
 +
-image::apikit-tutorial-3e0e4.png[height=633,width=600]
+image::apikit-tutorial-3e0e4.png[height=633,width=600][alt=Project Settings for a New Mule Project, including Project Name, Runtime, Maven Settings, Version Control System Support, APIkit Settings, and API Definition.]
```

Comment:
Careful here. All attributes for an image must be enclosed within the same brackets `[` `]`. Otherwise it produces a broken syntax:

`<img src="_images/apikit-tutorial-3e0e4.png" alt="apikit tutorial 3e0e4" width="600][alt=Project Settings for a New Mule Project" height="including Project Name">`


```suggestion
image::apikit-tutorial-3e0e4.png["Project Settings for a New Mule Project, including Project Name, Runtime, Maven Settings, Version Control System Support, APIkit Settings, and API Definition.",height=633,width=600]
```
Based on the [Asciidoctor documentation](https://docs.asciidoctor.org/asciidoc/latest/macros/images/), it's not necessary to declare the 'alt' attribute, but it must be the first attribute you add. Also, if the alt text has commas, you must enclose it in `"`.